src/main/java/frc/robot/RobotContainer.java

```diff
@@ -130,8 +135,8 @@ private void configureBindings() {
 
         drivetrain.setDefaultCommand( // Drivetrain will execute this command periodically
                 drivetrain.applyRequest(() -> {
-                            double originalX = -driverController.getLeftY();
-                            double originalY = -driverController.getLeftX();
+                            double originalX = -driverController.getLeftY() * (!FieldUtil.isAllianceBlue() ? -1 : 1);
```

Comment:
I'm not sure doing this is the best way to do it, tbh. I could be wrong. If this works then it's fine to leave it. Just as a nit, I would do `(FieldUtil.isAllianceBlue() ? 1 : -1)` instead of adding the `!` which makes it harder to read.